config.py

```diff
@@ -26,7 +26,7 @@ class Config(object):
     DEBUG = False
     ADMINS = frozenset([__YOUR_EMAIL__])
     TESTING = False
-    HOST = '127.0.0.1'
+    HOST = '0.0.0.0'
```

Comment:
I made a change on the Flask server to start with the flag `host=c.HOST`. That was what I needed in order for `Docker` to expose the app port as a system port. I like the idea of a `Docker` class, but I don't think it can be done in this pull request.